[2020-06-01 21:35:33,160] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_covid.multitask2 2020-05-30T06:00:00+00:00 [queued]>
[2020-06-01 21:35:33,168] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_covid.multitask2 2020-05-30T06:00:00+00:00 [queued]>
[2020-06-01 21:35:33,169] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 21:35:33,169] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-01 21:35:33,169] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 21:35:33,185] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): multitask2> on 2020-05-30T06:00:00+00:00
[2020-06-01 21:35:33,189] {standard_task_runner.py:53} INFO - Started process 24152 to run task
[2020-06-01 21:35:33,253] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_dag_covid.multitask2 2020-05-30T06:00:00+00:00 [running]> nineleaps
[2020-06-01 21:35:33,263] {logging_mixin.py:112} INFO - Hello-World
[2020-06-01 21:35:33,263] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-01 21:35:33,267] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=test_dag_covid, task_id=multitask2, execution_date=20200530T060000, start_date=20200601T160533, end_date=20200601T160533
[2020-06-01 21:35:43,156] {logging_mixin.py:112} INFO - [2020-06-01 21:35:43,155] {local_task_job.py:103} INFO - Task exited with return code 0
